components/baseline/_base.scss

```diff
@@ -7,19 +7,6 @@
 	background-color: #355688;
 }
 
-/* H1 short bold red underline */
-h1.gc-thickline {
```

Comment:
Comment/Note: We will need a technical note about that CSS class being obsoleted because its effect are now the new default. And that it is the new default, that is not a breaking change because any content that was specifying the `gc-thickline` will continue to appear and be rendered as it was before.

Not a requirement to get this PR merge, but that note must be added to GCWeb before the subsequent release. May be the best location for that note will be in the "index.json-ld" of the new "main-page-title" site component.